nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server.hpp

```diff
@@ -21,6 +21,10 @@
 #include <string>
 #include <vector>
 
+#include <boost/uuid/uuid.hpp>
```

Comment:
We need to link this / package.xml the dependency? Please only include just the boost libraries required (not all)